modules/imap/handler_modules.php

```diff
@@ -852,6 +852,14 @@ public function process() {
                     $msg['server_id'] = $form['imap_server_id'];
                     $msg['server_name'] = $details['name'];
                     $msg['folder'] = $form['folder'];
+                    $uid = $msg['uid'];
+                    $part = true;
+                    $max = 87;
+                    $msg_struct = $imap->get_message_structure($uid);
+                    $struct = $imap->search_bodystructure($msg_struct, array('imap_part_number' => $part));
+                    $msg_struct_current = array_shift($struct);
+                    $msg_text = $imap->get_message_content($uid, $part, $max, $msg_struct_current);
+                    $msg['preview_msg'] = $msg_text;
```

Comment:
It's displaying the `$subject` content, so why did you declare `$preview_msg` instead of just using `$subject`?

Reply:
It's not a subject content, it is the content of the message in preview mode.

can you see this capture of this [issue](https://github.com/cypht-org/cypht/issues/1159#issue-2459881679)